.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -87,7 +87,8 @@ jobs:
             qt_version: 5
             # FIXME: many tests fail to link
             # https://github.com/KDAB/cxx-qt/issues/111
-            ctest_args: --exclude-regex '^(cxx_qt_gen_cargo_tests|demo_threading_cargo_tests|example_qml.*|qml.*tests|test.*|reuse_lint|cpp_clang_format|.*valgrind)$'
+            # cxx_qt_lib_cargo_tests fails to run with a missing DLL error.
```

Comment:
can this commit come earlier in the chain ? As we should try to ensure that each commit passes tests so that when tools like `git bisect` are used things make sense. Or should i squash commit at the end ? (but i prefer to commit with the full chain normally...)